app/hypnos/CMakeLists.txt

```diff
@@ -7,8 +7,10 @@ list(APPEND BOARD_ROOT ${CMAKE_CURRENT_LIST_DIR}/..)
 list(APPEND DTS_ROOT ${CMAKE_CURRENT_LIST_DIR}/..)
 
 # Config files
-IF("$ENV{LOGGING}" STREQUAL "off")
-  message(STATUS "LOGGING will be disabled.")
+IF("$ENV{DFU}" STREQUAL "on")
```

Comment:
What about doing it independently ?
```
set(CONF_FILE ${CMAKE_CURRENT_LIST_DIR}/prj.c)
IF("$ENV{DFU}" STREQUAL "on")
  set(CONF_FILE ${CONF_FILE} ${CMAKE_CURRENT_LIST_DIR}/dfu.conf)
ENDIF()
IF("$ENV{LOGGING}" STREQUAL "on")
  set(CONF_FILE ${CONF_FILE} ${CMAKE_CURRENT_LIST_DIR}/logging.conf)
ENDIF()
```

Reply:
Any suggestion?

Reply:
It would be nice to be able to set any combination of envs/configs if we get more in the future.

Reply:
try it out. 
It sets the main confi file anyways and depending on the ENV variables it adds up the necessary files.

Reply:
OK! It will disable logging by default but that's OK I guess.

Reply:
@sdorre, I've made changes again and force-pushed.  Feel free to press merge if it looks alright to you now.